src/errors/permissionsError.js

```diff
@@ -0,0 +1,15 @@
+import BaseHttpError from './baseHttpError';
+import { LOW_PERMISSIONS } from '../constants/errors';
```

Comment:
You can hardcode default messages in this class, no need to separate them into constants

Reply:
Also use `http-status-codes` module to add `code` field on all of your errors.
Research and understand the difference between Authorization and Authentication (and status codes that these kind of errors need)